command-parser.js

```diff
@@ -634,7 +635,8 @@ let parse = exports.parse = function (message, room, user, connection, levelsDee
 	});
 
 	if (commandHandler) {
-		return context.run(commandHandler);
+		message = context.run(commandHandler);
+		if (!message || typeof message.then === 'function') return message;
```

Comment:
We still shouldn't return `message` in that case; we don't have a handler for that anymore...